### PR TITLE
title11

### PR DESCRIPTION
**Pull request by val-fed**
body

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
```
Comment (val-fed) [inline README.md]: i,umtyngbfv

File: README.md [hunk 2]
```diff
@@ -3,21 +3,21 @@ test1
...
 test6
 test7
 test8
 test9
 test10
-test11
+edit-test11
 test12
 test13
 test14
```
Comment (val-fed) [inline README.md]: iyujthygtfd

File: README.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
 test6
```
Comment (val-fed) [inline README.md]: loui,yjumhngbfvd

File: test.txt [hunk 4]
```diff
@@ -1,20 +1,17 @@
...
+
 test13
-test14
 test15
 test16
-test17
+edit-test17
 test18
 test19
 test20
```
Comment (val-fed) [inline test.txt]: iuuykuykk

File: README.md (hunk 1 above)
Comment (valerii-fed) [reply README.md]: it's ok

File: test.txt [hunk 5]
```diff
@@ -1,20 +1,17 @@
-test1
-test2
 test3
 test4
-test5
+edit
 test6
-test7
 test8
```
Comment (valerii-fed) [inline test.txt]: what do yo think?

**Pull request closed without merging**